backend/TraderX/src/main/java/cmpe451/group6/rest/transaction/dto/TransactionDTO.java

```diff
@@ -0,0 +1,53 @@
+package cmpe451.group6.rest.transaction.dto;
+
+import java.util.Date;
+
+import cmpe451.group6.rest.transaction.model.TransactionType;
+import io.swagger.annotations.ApiModelProperty;
+
+public class TransactionDTO {
+
+    @ApiModelProperty(position = 0, required = true)
+    private String equipment;
+    @ApiModelProperty(position = 1, required = true)
+    private TransactionType transactionType;
+    @ApiModelProperty(position = 2, required = true)
+    private Date createdAt;
+    @ApiModelProperty(position = 3, required = true)
+    private float count;
+
```

Comment:
Shouldnt we add "amount" field too?

Reply:
`count` looks like referring to `amount`. Might be changed to prevent confusion.